test/suite/test_timestamp03.py

```diff
@@ -227,21 +227,21 @@ def test_timestamp03(self):
 
         # Bump the oldest_timestamp, we're not going back...
         self.assertEqual(self.conn.query_timestamp(), timestamp_ret_str(100))
-        self.oldts = timestamp_str(100)
-        self.conn.set_timestamp('oldest_timestamp=' + self.oldts)
-        self.conn.set_timestamp('stable_timestamp=' + self.oldts)
+        old_ts = timestamp_str(100)
+        self.conn.set_timestamp('oldest_timestamp=' + old_ts)
+        self.conn.set_timestamp('stable_timestamp=' + old_ts)
```

Comment:
Could there be a test case to set the stable_timestamp older than the oldest timestamp. I think we should expect error in that case.

Reply:
yes, it returns error and is covered in other testing

Reply:
Ah, I see. It is there in the test_timestamp08.py added by @sulabhM in the changes for wt-3512. Thanks.